packages/protocol/contracts/stability/MedianDeltaBreaker.sol

```diff
@@ -91,22 +92,26 @@ contract MedianDeltaBreaker is IBreaker, UsingRegistry {
 
     address stableToken = IExchange(exchange).stable();
 
-    uint256 lastMedian = sortedOracles.lastMedianRate(stableToken);
-    if (lastMedian == 0) {
+    uint256 previousMedian = sortedOracles.previousMedianRate(stableToken);
+    if (previousMedian == 0) {
       // Last median will be 0 if this exchange is new and has not had a median update yet.
       return false;
     }
 
     (uint256 currentMedian, ) = sortedOracles.medianRate(stableToken);
 
     // Check if current median is within allowed threshold of last median
-    triggerBreaker = !isWithinThreshold(lastMedian, currentMedian, priceChangeThreshold.unwrap());
+    triggerBreaker = !isWithinThreshold(
+      previousMedian,
+      currentMedian,
+      priceChangeThreshold.unwrap()
```

Comment:
Does this need to be `unwrapped`? Can't we pass a FixidityFraction to the function?

Reply:
This doesn't even need to be here. Can be read directly in the function.